source/installation.txt

```diff
@@ -50,16 +49,25 @@ s390x systems running Red Hat Enterprise Linux 7.2+; and the following
 
    * - Debian (64-bit)
      - 8.1 or later ("Jessie")
-     -
+     - 9.1 or later ("Stretch")
 
    * - Ubuntu (64-bit)
      - 14.04 ("Trusty")
-     -
+     - 16.04 ("Xenial")
 
    * - SUSE Enterprise Linux (64-bit)
      - 12
      -
 
+|bi-short| also supports the following system architectures:
```

Comment:
Everything is covered but I'd make this consistent with SERVER's documentation; Linux on z Systems == s390x systems == IBM z Systems.

We use all 3 terms on this page.